Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/PIDDrive.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc.team199.robot.commands;
+
+import org.usfirst.frc.team199.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class PIDDrive extends Command {
+
+	double target;
+    public PIDDrive(double targ) {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    	target = targ;
+    	requires(Robot.dt);
```

Comment:
Once you create an interface for the Drivetrain (which will be the next step), you can great a local variable in this class of the type DrivetrainInterface so you don't have to call Robot.dt every time